pkgs/development/interpreters/python/wrap-python.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, python
+, makeSetupHook
+, makeWrapper }:
+
+with lib;
+
+makeSetupHook {
+      deps = makeWrapper;
+      substitutions.libPrefix = python.libPrefix;
+      substitutions.executable = python.interpreter;
+      substitutions.python = python;
+      substitutions.magicalSedExpression = let
```

Comment:
+1 to that. I have a rough idea of what it does, but my regex-fu isn't too great.